online-shop/src/app/mocks/products.mocks.ts

```diff
@@ -0,0 +1,20 @@
+import { Product } from "../modules/shared/types/products.types";
+
+export const MOCK_PRODUCTS: Product[] = [
+    {
+        "id": "7ed8b2fa-0d7f-4dab-a01e-5583841e0469",
+        "name": "Notebook Basic 15",
+        "category": "Laptops",
+        "image": "https://sapui5.hana.ondemand.com/test-resources/sap/ui/documentation/sdk/images/HT-1000.jpg",
+        "price": 956,
+        "description": "Notebook Basic 15 with 2,80 GHz quad core, 15\" LCD, 4 GB DDR3 RAM, 500 GB Hard Disc, Windows 8 Pro"
+    } as Product,
```

Comment:
_Suggestion_ : You can get rid of the casting (as Product), because you already define the type in line 3 and it will be recognized